java/restheart/pom.xml

```diff
@@ -7,10 +7,9 @@
   <packaging>jar</packaging>
 
   <properties>
-    <mongodb.version>4.2</mongodb.version>
     <maven.compiler.source>17</maven.compiler.source>
     <maven.compiler.target>17</maven.compiler.target>
-    <restheart.version>[5.4,5.5)</restheart.version>
+    <restheart.version>[6.0,7.0)</restheart.version>
```

Comment:
```suggestion
    <restheart.version>[6.0,6.1)</restheart.version>
```

Reply:
[6.0,7.0) also selects 7.0.0-SNAPSHOT. Although v7 has not yet been released, it is stable.
If you want to limit the test to stable releases, please use: `<restheart.version>[6.0,6.100)</restheart.version>`

Reply:
I prefer to have the last stable and published released. `renovate` should ensure updates

Reply:
Maybe
```suggestion
    <restheart.version>[6.5,6.6)</restheart.version>
```